src/witan/app/handler.clj

```diff
@@ -23,7 +23,7 @@
 
 (def LoginReturn
   (s/either {(s/required-key :token) s/Str
-             (s/required-key :id) s/Str}
+             (s/required-key :id) s/Uuid}
```

Comment:
The reason I used String instead of Uuid is because we still have to investigate which one we'll use + will this pass muster with a cassandra uuid?
(as in, it's not necessarily the same Java class)